client/index.html

```diff
@@ -9,6 +9,7 @@
 	<meta name="apple-mobile-web-app-status-bar-style" content="black-translucent">
 	<meta name="mobile-web-app-capable" content="yes">
 	<meta name="referrer" content="no-referrer">
+	<meta name="theme-color" content="#455164">
```

Comment:
I think that's something that will be properly dealt with with actual theme packages. Then, we could template the manifest and have it work better. But for now I don't think there's much to do, and anything looks better than white at least.